src/odin/ingestion/qlik/tables.py

```diff
@@ -0,0 +1,42 @@
+CUBIC_ODS_TABLES = [
```

Comment:
I thought we might have been able to control this through an object in S3. I'm thinking we might want to manipulate it without a code change at some point, and since we don't have a database, maybe an S3 object would be better. Just a thought.

Reply:
Also, what would be best for extracting the schema for all tables being sent by Cubic. We need to give data analysts the info to decide if they'd like to sync additional tables.